Demo/Demo/DemoMerchantAPI.swift

```diff
@@ -0,0 +1,78 @@
+import Foundation
+
+final class DemoMerchantAPI {
+
+    static let sharedService = DemoMerchantAPI()
+
+    private init() {}
```

Comment:
what do you think of making this function `init(environment: DemoSettings.Environment)` and add a `var environment`?
so we have an easy way to toggle between environments in our test app if we wanna :smile:

Reply:
I think since it's a singleton init, it wouldn't matter much what initialization parameters we add to it. Guessing this will eventually read from `DemoSettings` to get the current environment

Reply:
ah you're right!!!! Also agree with your suggestion on making `DemoSettings` a singleton too, it'd be nice having one place to store all settings :smile: